content/sql/shop-project/_index.md

```diff
@@ -22,6 +22,13 @@ title: Shop Database using sql
 - Instead of installing Postgres on your computer, you can launch it with a docker composition
 - MySQL is nice and lot's of people use it in industry, but it doesn't actually implement standard SQL, it sort of does it's own thing a bit. Postgres is a much more standard DB, and industry loooooves it
 
+## Structure:
+
+- Your repository should have a **.yml** file 
+- Your repository should have **.sql** files
+- The different SQL commands should be saved in different descriptive script files i.e.
+when the reviewer is looking for the commands you used for creating the database they should be able to navigate to a file named **create-database.sql**  , navigate to a file named **create-tables.sql** for the commands used to create the database tables , and navigate to the files that contain the commands used for inserting table records and querying the database
```

Comment:
This is good to point out. Can you please make this in bullet or list form? Just so it is clearer